src/database/config/config.json

```diff
@@ -3,22 +3,22 @@
     "username": "postgres",
     "password": "54321",
     "database": "ecommerce-app-cogito-db",
-    "host": "localhost",
-    "port": "8080",
+    "host": "postgres",
+    "port": "5432",
     "dialect": "postgres"
   },
   "test": {
     "username": "postgres",
     "password": "54321",
     "database": "ecommerce-app-cogito-db",
-    "host": "localhost",
+    "host": "postgres",
     "dialect": "postgres"
   },
   "production": {
     "username": "postgres",
     "password": "54321",
     "database": "ecommerce-app-cogito-db",
-    "host": "localhost",
+    "host": "postgres",
```

Comment:
Also here I would suggest that we could have different database names for the different use cases.